sample/android/src/main/java/com/example/sqldelight/hockey/HockeyApp.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.sqldelight.hockey
+
+import android.app.Application
+import com.example.sqldelight.hockey.data.Schema
+import com.example.sqldelight.hockey.data.driver
+import com.squareup.sqldelight.android.AndroidSqliteDriver
+
+class HockeyApp:Application(){
```

Comment:
This file needs formatted (not two-space indent, no space around colon, no space before opening curly)

Reply:
Reformatted. Missed that one pre-commit.